cfme/fixtures/v2v_fixtures.py

```diff
@@ -140,13 +146,49 @@ def create_tags(appliance, transformation_method):
     return tag1, tag2
 
 
-def set_conversion_instance_for_rhev(appliance, transformation_method, rhev_hosts):
-    """Assigning tags to conversion host.
+def vddk_url():
+    """Get vddk url from cfme_data"""
+    vddk_version = "v2v_vddk"
+    try:
+        vddk_urls = conf.cfme_data.basic_info.vddk_url
+    except (KeyError, AttributeError):
+        pytest.skip("VDDK URLs not found in cfme_data.basic_info")
+    url = vddk_urls.get(vddk_version)
+
+    if url is None:
+        pytest.skip("VDDK {} is unavailable, skipping test".format(vddk_version))
+    return url
+
+
+def configure_conversion_host_ui(appliance, target_provider, hostname, default,
+                                 conv_host_key, transformation_method,
+                                 vmware_ssh_key, osp_cert_switch=None, osp_ca_cert=None):
+
+    conv_host_collection = appliance.collections.v2v_conversion_hosts
+    conv_host = conv_host_collection.create(
+        target_provider=target_provider,
+        provider_name=target_provider.name,
+        cluster=get_data(target_provider, "clusters", default),
+        hostname=hostname,
+        conv_host_key=conv_host_key,
+        transformation_method=transformation_method,
+        vddk_library_path=vddk_url(),
+        vmware_ssh_key=vmware_ssh_key,
+        osp_cert_switch=osp_cert_switch,
+        osp_ca_cert=osp_ca_cert
+    )
+    if not conv_host.is_host_configured:
+        pytest.skip("Failed to set conversion host/instance: {}".format(hostname))
```

Comment:
This function is not decorated as a fixture, so its not appropriate to use `pytest` here, as this function could be imported and used outside of a test.

It should raise an exception, return a boolean indicating success, or be directly decorated as a fixture. In the first two cases, you could write an actual fixture method that catches the exception and skips.

Reply:
I realize this is a bad practice throughout this file, with many functions not decorated as fixtures and calling `pytest` methods.

One option is renaming them as 'private' methods, to indicate that they aren't intended for use outside this module, and then only call them locally here in actual fixture methods.

Reply:
Made methods private